hail/benchmark/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:18.04
+
+ENV LANG C.UTF-8
+
+ARG WHEEL
+
+RUN apt-get update && \
+  apt-get -y install \
+    git \
+    htop \
+    unzip bzip2 zip tar \
+    wget curl \
+    rsync \
+    emacs25-nox \
+    mysql-client \
+    xsltproc pandoc \
+    jq \
+    openjdk-8-jdk-headless \
+    python \
+    python3 python3-pip && \
+  rm -rf /var/lib/apt/lists/*
+
+
+COPY requirements.txt .
+RUN pip3 install -r requirements.txt
```

Comment:
this bit is not necessary (the wheel will install these dependencies), but it takes the size of  wheel layer from 1G (including pyspark and all deps) to 30M, making iteration much faster.